### PR TITLE
Enabling velocity mode

### PR DESCRIPTION
**Pull request by livanov93**
The aim of the PR is to enable velocity based control within the UR driver.

TODOs:

- enable command interface switching 
- improve command interface detection logic
- add velocity command in the update loop

### CODE REVIEW

File: ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+# limitations under the License.
+#
+# Author: Lovro Ivanov
+#
+# Description: After a robot has been loaded, if velocity controller is up
+# and running, moveit servo will take control of the robot by  gradually
+# increasing z-component of the robot_link_command_frame
+
+import os
+import yaml
```
[the remark this answers is not in the document]
Comment (livanov93) [reply ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py:22]: They are needed by load_yaml which is still in use even after [cleanup](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/146/commits/c4f8b02ffff95973e00a2ea4b2d9065c852e3770).

File: ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+            " ",
+            "prefix:=",
+            prefix,
+            " ",
+        ]
+    )
+    robot_description_semantic = {"robot_description_semantic": robot_description_semantic_content}
+
+    # Get parameters for the Pose Tracking node
+    pose_tracking_yaml = load_yaml("moveit_servo", "config/pose_tracking_settings.yaml")
```
[the remark this answers is not in the document]
Comment (livanov93) [reply ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py:254]: Leaving load_yaml even after [cleanup](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/146/commits/c4f8b02ffff95973e00a2ea4b2d9065c852e3770).

File: ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    servo_params = {"moveit_servo": servo_yaml}
+
+    kinematics_yaml = load_yaml("ur_moveit_config", "config/kinematics.yaml")
+
+    joint_limits_yaml = {"robot_description_planning": joint_limit_params}
+
+    # RViz
+    rviz_config_file = (
+        get_package_share_directory("moveit_servo") + "/config/demo_rviz_pose_tracking.rviz"
+    )
```
[the remark this answers is not in the document]
Comment (livanov93) [reply ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py]: [c4f8b02](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/146/commits/c4f8b02ffff95973e00a2ea4b2d9065c852e3770)

File: ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+    package_path = get_package_share_directory(package_name)
+    absolute_file_path = os.path.join(package_path, file_path)
+
+    try:
+        with open(absolute_file_path) as file:
+            return file.read()
+    except OSError:  # parent of IOError, OSError *and* WindowsError where available
+        return None
+
+
```
[the remark this answers is not in the document]
Comment (livanov93) [reply ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py]: [c4f8b02](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/146/commits/c4f8b02ffff95973e00a2ea4b2d9065c852e3770)

**Review by destogl — APPROVED** [review]
Thanks for following-up the comments.

**Pull request merged**